src/config.rs

```diff
@@ -3,14 +3,27 @@ use {
   clap::{App, AppSettings, Arg, ArgGroup, ArgMatches, ArgSettings},
 };
 
-// These three strings should be kept in sync:
-pub(crate) const CHOOSER_DEFAULT: &str =
-  "fzf --multi --preview 'just --unstable --color always --show {}'";
+// These two strings, and the two in function chooser_default below should be kept in sync:
 pub(crate) const CHOOSER_ENVIRONMENT_KEY: &str = "JUST_CHOOSER";
 pub(crate) const CHOOSE_HELP: &str = "Select one or more recipes to run using a binary. If \
                                       `--chooser` is not passed the chooser defaults to the value \
                                       of $JUST_CHOOSER, falling back to `fzf`";
 
+// Return the string for the default chooser. This is a function and not a const, because we want
+// to edit the preview command and pass it the right justfile path for the target.
+pub(crate) fn chooser_default(justfile: &Justfile) -> OsString {
```

Comment:
```suggestion
pub(crate) fn chooser_default(justfile: &Path) -> OsString {
```

Reply:
Right, it also makes the function simpler, no need to check the Option value.

Reply:
sorry, wrong GH account - I'm deleting this

Reply:
Right, it also makes the function simpler, no need to check the Option value.